project-base/app/src/Controller/Admin/OrderController.php

```
@@ -0,0 +1,72 @@
+     */
+    #[Override]
+    protected function configureDatagrid(Datagrid $datagrid): void
+    {
+        $datagrid->add('number');
+        $datagrid->add('deleted');
+
+        $datagrid->setDefaultOrder('deleted', OrderingEnum::DESC);
+
+        $datagrid->add('note', [
+            'virtual' => true,
+            'transform' => function ($value, $row) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$value'. (undefined)

(UnusedFormalParameter)